README.md

```diff
@@ -557,9 +557,9 @@ https://github.com/offensive-security/exploit-database/tree/master/platforms/lin
 
 https://bugs.chromium.org/p/project-zero/issues/list?can=1&q=linux+kernel&colspec=ID+Type+Status+Priority+Milestone+Owner+Summary&cells=ids
 
-http://vulnfactory.org/exploits/
+http://vulnfactory.org/exploits/ (too old 2010 2011)
```

Comment:
A lot of the existing exploits are quite old, they are still somewhat useful to keep for learning/historical purposes. I don't think this comment is useful just by itself.